ooc-cli.use

```diff
@@ -0,0 +1,6 @@
+Name: ooc CLI
+Description: CLI helpers
+SourcePath: source/cli
+Requires: ooc-collections
```

Comment:
Because people are never going to remember to keep these files up to date, otherwise.